### PR TITLE
Fix indexing issues caused by ampersand-collection@1.4.2

### PR DESCRIPTION
**Pull request by latentflip**

### CODE REVIEW

File: ampersand-filtered-subcollection.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ var slice = Array.prototype.slice;
 
 function FilteredCollection(collection, spec) {
     this.collection = collection;
-    this.indexes = collection.indexes;
+    this.indexes = collection.indexes || [];
```
Comment (wraithgar) [inline ampersand-filtered-subcollection.js]: This `|| []` is done in the `reset` method in collection. idk if we care about that level of consistency.

**Comment by wraithgar** [comment]
+1 pending food-for-thought nitpick

File: ampersand-filtered-subcollection.js (hunk 1 above)
Comment (latentflip) [inline ampersand-filtered-subcollection.js]: IMO it's flawed that it doesn't default to an empty array on initialize in collection tbh.

**Comment by legastero** [comment]
+1

**Comment by wraithgar** [comment]
published as v1.0.1

**Pull request merged**